### DISCUSSION TITLE
Accessing the elements of the full triangle

### DISCUSSION THREAD

**Discussion by AmyRandelhoff** [casact/chainladder-python#535] [General]
**How can we extract the predicted values from the full triangle once the basic chain ladder model has been fit?**

So, once we have performed this step:
model = cl.Chainladder().fit(data)

model.full_triangle_ - this gives the full triangle, but I want to access the predicted values (store them in an array or dataframe to perform analysis on them)

model.ultimate_ - gives only the predicted ultimate values and not the other predicted values for development periods between the valuation date and the ultimate date

I tried using the to.frame() method but that does not seem to work for  "<class 'chainladder.methods.chainladder.Chainladder'>"  

_print(model.to_frame())
AttributeError: 'Chainladder' object has no attribute 'to_frame'_

Thanks for any help!

**Comment by jbogaardt** [comment]
Could you do `model.full_triangle_.to_frame()` to resolve your example?
